MainActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.soumyajyoti.firstapplication;
```

Comment:
Keep it with professional name, not soumyajyoti.firstapplication.
com.helloworld.firstapp